File: solutions/lua/raindrops/1/raindrops.lua
return function(n)
  ret = ""
  if n % 3 == 0 then ret = "Pling" end
  if n % 5 == 0 then ret = ret .. "Plang" end
  if n % 7 == 0 then ret = ret .. "Plong" end
  if ret == "" then return tostring(n) else return ret end
end

File: solutions/go/raindrops/1/raindrops.go
package raindrops

import "strconv"

// Convert returns a string containing Pling, Plang and/or Plong depending if the given number is divisible by 3, 5 and/or 7.
func Convert(in int) string {
	result := ""
	if in%3 == 0 {
		result += "Pling"
	}
	if in%5 == 0 {
		result += "Plang"
	}
	if in%7 == 0 {
		result += "Plong"
	}
	if len(result) > 0 {
		return result
	}
	return strconv.Itoa(in)
}
